### DISCUSSION TITLE
글의 분위기와 경칭 사용 방안 논의

### DISCUSSION THREAD

**Discussion by potatogim** [go101korea/go101#59] [General]
본격적으로 번역하기에 앞서 전반적인 분위기와 경칭 사용 방안에 대해 논의해봤으면 합니다.

1. 분위기
    1. 구어체 중심의 가벼운 느낌으로...? (예: 겠어요, 해볼게요 등)
    2. 문어체 중심의 격식있는 느낌으로...? (예: 겠습니다, 해보겠습니다 등)
3. 경칭 사용 방안
    1. 인명에 대한 경칭: 씨? 님?

**Comment by AngeliKaroS** [comment]
개인적으로 전체적인 분위기는 가볍게 가는게 좋지 않을까 싶습니다
경칭은.. 어차피 닉으로 불릴테니 ~님이 낫지 않을까 생각합니다

**Reply by AngeliKaroS** [reply]
아 이게 여기 분위기가 아니라 글 전체적인 분위기였군여..

글 분위기.. 반드시 경어체를 써야하나용? ㅎ

**Reply by potatogim** [reply]
아무래도 개인 블로그에 쓰는 글이 아니고 공식적인 문서로서 여러 독자를 대상으로 하기 때문에 경어체를 쓰는게 좋을 것 같네요~!

**Comment by kjy8901** [comment]
경칭사용 방안
- 사내에 있는 책 몇권을 확인해봤는데 아래 같은 케이스가 보이네요
  - 그냥 영어로 사용하는 케이스: taylor -> taylor
  - 영어 -> 한글 번역: taylor -> 테일러
  - 국내 책의 경우 : 홍길동님
 
분위기
- 제 생각에는 정보를 전달하는 글이다보니 구어체 보다는 문어체가 더 좋을 것 같습니다

**Comment by hspark0582** [comment]
- 문어체를 기본으로 상황에 따라 가끔 구어체를 섞는 정도가 좋을 것 같습니다.
- ~씨의 경우 기본적으로 독자/청자에게 경어를 사용하는 상황에서 사용하는 압존법의 개념으로 알고 있습니다. 책의 분위기 상 경어를 사용할 것은 아니어보이니 ~님을 사용하거나 경칭을 생략하는 것이 적당해 보입니다.

**Comment by potatogim** [comment]
그럼 우선 분위기 자체는 일반적인 사례들과 유사하게 격식을 갖추되 너무 무겁게 가겨가지 않도록 해보죠~ ㅎㅎ

경칭의 경우에는 님은 너무 격식을 높이는 느낌이라 중립적인 존칭인 씨 정도가 적당해보이네요.

https://ko.dict.naver.com/#/entry/koko/1db514b8b9be4f23a6a040a7b498805b

**Reply by AngeliKaroS** [reply]
음 그러면 ~이다 ~다 이런 표현은 수정 하는게 좋을까요..?

**Reply by potatogim** [reply]
그러는게 좋을 것 같네요~!
